docs/.vuepress/config.js

```diff
@@ -78,6 +78,7 @@ module.exports = {
                 title: 'Howto Series',
                 path: '/series/',
                 children: [
+		  '/series/LAMP-server',
```

Comment:
I don't think there is a pipeline to `lint` stuff, but this seems to be missing a tab.

Reply:
Thanks, I fixed it.